docs/adguard-for-android/solving-problems/extending-restricted-settings.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Problems caused by extending restricted settings
+sidebar_position: 18
+---
+
```

Comment:
There's an INFO plaque missing that's present in all other articles about AdGuard for Android